### ISSUE TITLE
Pre-Requisites not mentioned accurately

### ISSUE THREAD

**Issue by NisantaSahoo** [openebs/openebs#492]
## Is this a BUG REPORT or FEATURE REQUEST?
BUG REPORT 

**What happened**:
Not able to understand the pre-requisites for Setting up OpenEBS with Kubernetes using Minikube where it has been mentioned that Machine Type - (Minimum 4 vCPUs)

**What you expected to happen**:
It could have been explained well
**How to reproduce it (as minimally and precisely as possible)**:
may be in simpler terms

**Anything else we need to know?**:

**Environment**:
- kubectl get nodes
- kubectl get pods --all-namespaces
- kubectl get services
- kubectl get sc
- kubectl get pv
- kubectl get pvc
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by vharsh** [comment]
It'll be good to have a section/wiki somewhere where we enlist the processes for installing dependencies.

**Comment by aj07** [comment]
@NisantaSahoo  I would like to take this. Please, assign it to me.

**Comment by vharsh** [comment]
@aj07 All yours :+1: 
@aj07 Do share your thoughts on this here.

**Comment by aj07** [comment]
IS your cafe for hacktoberfest opened today?

**Comment by vharsh** [comment]
@aj07 Yes, we are open from 09:30

**Comment by yudaykiran** [comment]
The Docs for Minikube are now available at: https://docs.openebs.io/docs/devsolutions.html
The docs have to be updated in a way that the end user is able to understand.

**Comment by mahebbar** [comment]
The updated 
prerequisites are here https://docs.openebs.io/docs/next/prerequisites.html
minikube - https://docs.openebs.io/docs/next/supportedplatforms.html#native-kubernetes-and-minikube
